src/Tizen.NUI.Components/Controls/Navigation/Navigator.cs

```diff
@@ -425,11 +431,14 @@ public static void ShowAlertDialog(string title = null, string message = null, s
             var window = NUIApplication.GetDefaultWindow();
             var defaultNavigator = window.GetDefaultNavigator();
 
-            var dialog = new AlertDialog(title, message, positiveButtonText, positiveButtonClickedHandler, negativeButtonText, negativeButtonClickedHandler);
-            SetDialogScrim(dialog);
-
-            var dialogPage = new Page(dialog);
-            defaultNavigator.Push(dialogPage);
+            using (var dialog = new AlertDialog(title, message, positiveButtonText, positiveButtonClickedHandler, negativeButtonText, negativeButtonClickedHandler))
+            {
+                SetDialogScrim(dialog);
+                using (var dialogPage = new Page(dialog))
```

Comment:
using(){ } should not be used in this case since Navigator.Push does not clone its parameter.